drivers/spi/Kconfig

```diff
@@ -188,6 +203,44 @@ config SPI_2_CS_GPIO_PIN
 	depends on SPI_2 && SPI_CS_GPIO
 	default 0
 
+config SPI_3
+	bool
+	prompt "SPI port 3"
+	default n
+	help
+	  Enable SPI controller port 3.
+
+config SPI_3_NAME
+	string
+	prompt "SPI port 3 device name"
+	depends on SPI_3
+	default "SPI_3"
+
+config SPI_3_IRQ_PRI
+	int
+	prompt "Port 3 interrupt priority"
+	depends on SPI_3
+
+config SPI_3_DEFAULT_CFG
+	hex "Port 3 default configuration"
+	depends on SPI_3
+	default 0x80
+
+config SPI_3_DEFAULT_BAUD_RATE
+	int "Port 3 default baud rate"
+	depends on SPI_3
+	default 500000
+
```

Comment:
remove these 2 options above as you don't seem to use it. This was introduced by qmsi spi shim driver, for no good reason actually (and will become obsolete as soon as new spi API gets in)